pkg/operator/status/inertia.go

```diff
@@ -0,0 +1,56 @@
+package status
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+// Inertia returns the inertial duration for the given condition.
+type Inertia func(condition operatorv1.OperatorCondition) time.Duration
```

Comment:
Thoughts on pass-by-value here?  A pointer `condition` would be cheaper, and it seems unlikely that an operator would abuse it to mutate the passed condition.

Reply:
Without analysis, there is no guarantee it would be cheaper. I would prefer the pass-by-value as it correlates with the expected mutability of the parameter.